packages/core/admin/admin/src/pages/AuthPage/components/Register/index.js

```diff
@@ -98,6 +98,10 @@ const Register = ({ authType, fieldsToDisable, noSignin, onSubmit, schema }) =>
 
       if (!['password', 'confirmPassword'].includes(key) && typeof value === 'string') {
         normalizedvalue = normalizedvalue.trim();
+
+        if (key === 'lastname') {
```

Comment:
Don't we have a way to detect if a field is required or nullable that we can use instead of manually checking a specific field?

Reply:
I don't know - can yup do that?

Reply:
If the expected behaviour for a front-end nullable text input is that it should send as null, we should have that universally. I have no idea if that's the case for the content manager, but I assume/hope there's a standard way it's already done there that we could apply here.

For now this is acceptable to get lastname working, but it would be nice to know that if we add another field some day we don't have to add another exception for it if it's not too complicated to add.

Reply:
> but I assume/hope there's a standard way it's already done there that we could apply here.

There is not at the moment. The reason being: React does not like `null` values as input and will complain that you have to render empty string as value for text inputs. That makes the situation a bit tricky everywhere at the moment, because whenever we receive `null` from the API we have to coerce it to empty string and when you submit back to `null` OR empty string, because both cases are valid.

The CM has a lot of logic around that and currently there is no way to handle this everywhere. As this is specific CMS behavior we also can not build that into the design-system. In v5 we will think about that for sure, but right now this is IMO out of scope.